chkcryptoboot_hook

```diff
@@ -19,9 +20,35 @@ check_mbr ()
     fi
 }
 
+check_core ()
+{
+    if [ -n "$BOOTDISK" -a -s /etc/chkcryptoboot/disk-head.sha512 ]; then
+	resolve_device "$BOOTDISK"
+	if [ -n "$BOOT_PARTITION" -a -s /etc/chkcryptoboot/grub-core.sha512 ]; then
+	    dd if=$BOOT_PARTITION of=/etc/chkcryptoboot/grub-core bs=512 count=62
+	else
+	    dd if=$BOOTDISK of=/etc/chkcryptoboot/grub-core skip=1 bs=512 count=62
+	fi
+	cd /etc/chkcryptoboot
+	sha512sum -c disk-head.sha512
```

Comment:
Just noticed this typo here, it should be `sha512sum -c grub-core.sha512`. I will change this tonight.